.github/workflows/ci_macos_arm64_clang.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - macOS arm64 clang
+
+on:
+  schedule:
+    # Weekday mornings at 09:15 UTC = 01:15 PST (UTC - 8).
+    - cron: "15 9 * * 1-5"
+  workflow_dispatch:
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit). The workflow name is prepended to avoid conflicts between
+  # different workflows.
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  macos_arm64_clang:
```

Comment:
Please add the "Post to Discord on Failure" step at the end of this job

https://github.com/iree-org/iree/blob/6d6bd6e86a751611b44ec8c5fab8987cfff97843/.github/workflows/ci_macos_x64_clang.yml#L55-L66